server/queries/classQueries/recommendedQueries.ts

```diff
@@ -0,0 +1,9 @@
+import { Recommendation } from '../../models';
+
+const recommendedQueries = async (classId: Number) => Recommendation
+  .findAll({
+    attributes: ['id', 'material_link', 'description'],
```

Comment:
Sequelize will return all the attributes, so if you need them all, then no need to write them.